lib/takeaway.rb

```diff
@@ -0,0 +1,19 @@
+require 'menu.rb'
+require 'order.rb'
+class Takeaway
+  attr_reader :menu, :place_order
```

Comment:
You have an attribute reader here for an attribute that doesn't exist (`@place_order`). I'm guessing this appeared in order to pass the last test in takeaway_spec.rb?